setup.cfg

```diff
@@ -0,0 +1,3 @@
+[metadata]
+# compatibility until https://github.com/frappe/bench/pull/1502 is more adopted
```

Comment:
As long as this is required I can't merge it. Bench has to be backward and forward compatible. We don't do major versions for bench for a reason.

Reply:
Ok, understood. I checked again and I think I found a different compat mechanism that I have overlooked before that is leaner on the change and fully covered by releaserc automation: d59970d138f24639fd901e68d49fd442b8aefc0c

Reply:
I hope that's ok now, it allows us to eventually deprecated in the far distant future without any need to immediately take any action, while at the same time using a bit better practices (from packaging perspective).

Reply:
I hope that's ok now, it allows us to eventually deprecated in the far distant future without any need to immediately take any action, while at the same time using a bit better practices (from packaging perspective).

Reply:
https://github.com/frappe/frappe/pull/23231/commits/ea74a92e4d37bc52d6e62036b67171ea5c0cbda1 is now the only other way I could come up with.

Another alternative could be to merge https://github.com/frappe/bench/pull/1502 and wait for a _really_ long period (a year or two?) before following up with this changeset, then without compat.